tests/tacacs/test_ro_disk.py

```diff
@@ -64,12 +84,56 @@ def do_reboot(duthost, localhost, dutip, rw_user, rw_pass):
     wait(wait_time, msg="Wait {} seconds for system to be stable.".format(wait_time))
 
 
+def set_syslog_dest(duthost):
+    syslog_dir = "/run/mount/logs/"
+    res = duthost.shell("mkdir -p {}".format(syslog_dir))
+    assert res["rc"] == 0, "failed to create {}".format(syslog_dir)
```

Comment:
This assert is unnecessary because when rc is not 0, the duthost.shell call will raise an exception unless argument `ignore_module_errors=True` is supplied.